streaming_stt_nemo/__init__.py

```diff
@@ -12,6 +12,8 @@ class Model:
 
 
     def __init__(self, lang="en"):
+        # OVOS uses 'en-us' so this hacks that to work until dialects are supported
+        lang = lang.split('-')[0]
```

Comment:
This logic is OVOS dependent but this package is completely independent

So please migrate this change to https://github.com/NeonGeckoCom/streaming-stt-nemo/pull/13

Reply:
Do you mean move to the plugin here? https://github.com/NeonGeckoCom/neon-stt-plugin-nemo/pull/7

Reply:
It does not work from there. The lang is passed from ovos somewhere. That is why I put it here. If dialects are ever supported,  this could go away

Reply:
We could parse it where we init the model https://github.com/builderjer/neon-stt-plugin-nemo/blob/dev/neon_stt_plugin_nemo/__init__.py#L52

Reply:
This worked and should now be good